server/helpers/ffmpeg-utils.ts

```diff
@@ -106,6 +116,10 @@ type TranscodeOptions = {
 
 function transcode (options: TranscodeOptions) {
   return new Promise<void>(async (res, rej) => {
+    if (options.resolution === undefined) {
+      return rej('Invalid video file')
```

Comment:
rather than fail on undefined resolution (something which we didn't do so far), I'd rather keep the check for `options.resolution !== undefined` below and make 

```typescript
if (targetBitrate) command.outputOptions([`-maxrate ${ targetBitrate }`, `-bufsize ${ targetBitrate * 2 }`])
```

Reply:
I thought this would be fine, because there is probably no case where ffprobe can't discover the resolution, but still can transcode the video. But I will change it.